### PR TITLE
Grammar, Druid Master + Fencer Master, Item Descriptions

### PR DESCRIPTION
**Pull request by vtange**
Not a whole lot of lines, just responding to users' reports at the forums.

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -6069,9 +6069,9 @@ ITEM_20150918_006068	Krag Pike
 ITEM_20150918_006069	Pilgrim Pike
 ITEM_20150918_006070	Istora Pike
 ITEM_20150918_006071	Arde Dagger
-ITEM_20150918_006072	
+ITEM_20150918_006072	The fire sprouting from this blade doesn't affect the user, but may instead catch onto enemies.
 ITEM_20150918_006073	Karacha Dagger
-ITEM_20150918_006074	
+ITEM_20150918_006074	 - Every attack from this dagger has a 2% chance of inflicting poison on an enemy that deals 6165 damage to it over the course of 10 seconds.
```
Comment (Sunaries) [inline ITEM.tsv]: I have: 

[Poison Effect] 2% chance to inflict 6165 damage every second over 10 seconds to an enemy

Comment (vtange) [inline ITEM.tsv]: That's a HUGE amount of damage, You sure?

**Comment by Sunaries** [comment]
@imcgames 

단검으로 공격 시 2% 확률로 적에게 1초 간격으로 10초간 6165 피해를 주는 중독 효과 발생

↑ This item effect is currently translated as: "[Poison Effect] 2% chance to inflict a total of 6165 damage to an enemy over 10 seconds." 

The wording in Korean is hard. Does this mean 6165 damage **per second** over the course of 10 seconds **or** a total of 6165 damage?

**Comment by vtange** [comment]
Basically, is the total damage 6165 or 61650?

**Comment by imcgames** [comment]
@Sunaries  the second case, a total of 6165 damage.

**Comment by vtange** [comment]
Good to Merge then :)

**Comment by imcgames** [comment]
:+1:

**Pull request merged**